modules/pel/peltool/README.md

```diff
@@ -0,0 +1,83 @@
+# peltool Usage
+
+To see all of peltool's capabilities, use the help option `peltool.py --help`.
+
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
+          e.g. `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`,
+               `peltool.py -p <pel_dir> -a`
+
+#### List PELs
```

Comment:
it gives a warning here about skipping indent levels, so I guess this should be a '##'